Tools/Scripts/run-javascriptcore-tests

```diff
@@ -833,7 +833,7 @@ sub runJSCStressTests
         "/usr/bin/env", "ruby", "Tools/Scripts/run-jsc-stress-tests",
         "-j", jscPath($productDir), "-o", $jscStressResultsDir, "--arch", $archs);
 
-    if (nativeArchitecture($remotes) ne $archs) {
+    if (nativeArchitecture($remotes) ne $archs && $archs ne "arm") {
```

Comment:
I think it would be good to have a comment explaining why this condition exists.